[2025-08-18T14:06:25.848+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-18T14:06:26.045+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.validate_transform manual__2025-08-18T13:52:02.048845+00:00 [queued]>
[2025-08-18T14:06:26.106+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_pipeline_batch_data.validate_transform manual__2025-08-18T13:52:02.048845+00:00 [queued]>
[2025-08-18T14:06:26.108+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2025-08-18T14:06:26.216+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): validate_transform> on 2025-08-18 13:52:02.048845+00:00
[2025-08-18T14:06:26.245+0000] {standard_task_runner.py:63} INFO - Started process 5928 to run task
[2025-08-18T14:06:26.263+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'youtube_pipeline_batch_data', 'validate_transform', 'manual__2025-08-18T13:52:02.048845+00:00', '--job-id', '375', '--raw', '--subdir', 'DAGS_FOLDER/youtube_pipeline.py', '--cfg-path', '/tmp/tmpn2y0unyh']
[2025-08-18T14:06:26.276+0000] {standard_task_runner.py:91} INFO - Job 375: Subtask validate_transform
[2025-08-18T14:06:26.556+0000] {task_command.py:426} INFO - Running <TaskInstance: youtube_pipeline_batch_data.validate_transform manual__2025-08-18T13:52:02.048845+00:00 [running]> on host 47205959681c
[2025-08-18T14:06:27.382+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='youtube_pipeline_batch_data' AIRFLOW_CTX_TASK_ID='validate_transform' AIRFLOW_CTX_EXECUTION_DATE='2025-08-18T13:52:02.048845+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-18T13:52:02.048845+00:00'
[2025-08-18T14:06:27.387+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-18T14:06:27.489+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-18T14:06:27.494+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n        ROW_COUNT=$(docker exec namenode hdfs dfs -count /storage/hdfs/processed/golden_dataset | awk \'{print $2}\')\n        if [ "$ROW_COUNT" -lt 1 ]; then\n            echo "Transform failed: no rows in golden dataset" && exit 1\n        else\n            echo "Transform OK ($ROW_COUNT rows)"\n        fi\n\n        ']
[2025-08-18T14:06:27.574+0000] {subprocess.py:86} INFO - Output:
[2025-08-18T14:06:41.009+0000] {subprocess.py:93} INFO - Transform OK (112 rows)
[2025-08-18T14:06:41.009+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-18T14:06:41.010+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-18T14:06:41.109+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=youtube_pipeline_batch_data, task_id=validate_transform, execution_date=20250818T135202, start_date=20250818T140626, end_date=20250818T140641
[2025-08-18T14:06:41.173+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-18T14:06:41.455+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-18T14:06:41.459+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
